hash.k

```diff
@@ -0,0 +1,20 @@
+module HASH
+    imports STRING-SYNTAX
+    imports BYTES
```

Comment:
We probably shouldn't import this, otherwise we will get confused between MBytes (michelson bytes) and the hooked Bytes sort.  If we only use the string variants of these instructions, we should only import those variants.

Reply:
We import several byte conversion routines anyway --- this is due to the fact that essentially the string that I need to pass in is actually a bytestring --- the string input type is misleading. Unless we recreate the byte conversion routine, we need to import BYTES.